src/main/java/org/sonatype/repository/helm/internal/metadata/IndexYamlAbsoluteUrlRewriter.java

```diff
@@ -108,6 +108,10 @@ private Event maybeSetAbsoluteUrlAsRelative(ScalarEvent scalarEvent) throws Malf
       URI uri = new URIBuilder(oldUrl).build();
       if (uri.isAbsolute()) {
         String fileName = uri.getPath();
+        // Strip leading slash
+        if (!fileName.isEmpty()) {
```

Comment:
This in theory could throw a NPE (if fileName is null), you might do `null != fileName`. Given getPath can return null, you might want to do that!